match2/wikis/starcraft2/match_group_legacy.lua

```diff
@@ -41,6 +42,12 @@ function Legacy.get(frame)
 	newArgs.id = bracketid
 	newArgs["1"] = templateid
 
+	local storage = _args.store
+	if storage == '' or storage == nil then
+		storage = Variables.varDefault('disable_SMW_storage') == 'true' and 'false'
+	end
+	newArgs.store = storage
+
```

Comment:
This uses single quotes while the rest of the file uses double quotes. Will need a PR to fix the rest of the file.

Reply:
see #90

Reply:
Thank you!